data/journal_check/bug_refs.json

```diff
@@ -312,6 +312,13 @@
         },
         "type": "ignore"
     },
+    "health-check": {
```

Comment:
The should be a unique name - the old every with the same babe is being overwritten

Reply:
I should stop opening PRs that early I suppose.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15489

will create VRs for old and new strings